### ISSUE TITLE
Logging on SuperMUC-NG with cmake

### ISSUE THREAD

**Issue by sebwolf-de** [SeisSol/SeisSol#173]
**Describe the bug**
When I build with cmake, the log is not filtered, i.e. I get log output from all ranks. In addition the log info from the cpp parts are not outputted.

**Expected behavior**
I expect to see all logs from the fortran and cpp part emitted by rank 0.

**To Reproduce**
1. Current master a2e2e371031d35e60d39b55bc0559829904bbd6b
2. ccmake output:
 ```
ARCH                             dskx 
 ASAGI                            OFF 
 CMAKE_BUILD_TYPE                 Release 
 CMAKE_INSTALL_PREFIX             /usr/local 
 COMMTHREAD                       ON 
 CXXTEST_INCLUDE_DIR              /dss/dsshome1/03/ga35kum3/SeisSol/build/submodules/cxxtest 
 CXXTEST_PERL_TESTGEN_EXECUTABL   CXXTEST_PERL_TESTGEN_EXECUTABLE-NOTFOUND 
 CXXTEST_PYTHON_TESTGEN_EXECUTA   CXXTEST_PYTHON_TESTGEN_EXECUTABLE-NOTFOUND 
 DYNAMIC_RUPTURE_METHOD           quadrature 
 EQUATIONS                        viscoelastic2 
 HDF5                             ON 
 HDF5_C_LIBRARY_dl                /usr/lib64/libdl.so 
 HDF5_C_LIBRARY_hdf5              /dss/dsshome1/lrz/sys/spack/release/19.1/opt/x86_avx512/hdf5/1.8.20-in
 HDF5_C_LIBRARY_m                 /usr/lib64/libm.so 
 HDF5_C_LIBRARY_pthread           /usr/lib64/libpthread.so 
 HDF5_C_LIBRARY_sz                /dss/dsshome1/lrz/sys/spack/release/19.1/opt/x86_avx512/libszip/2.1.1-
 HDF5_C_LIBRARY_z                 /dss/dsshome1/lrz/sys/spack/release/19.1/opt/x86_avx512/zlib/1.2.11-gc
 LOG_LEVEL                        INFO 
 MEMKIND                          OFF 
 METIS                            ON 
 MPI                              ON 
 NETCDF                           ON 
 NUMBER_OF_MECHANISMS             3 
 NUMBER_OF_MULTIPLE_SIMULATIONS   1 
 OPENMP                           ON 
 ORDER                            6 
 PLASTICITY                       OFF 
 PLASTICITY_METHOD                nb 
 SIONLIB                          OFF 
 yaml-cpp_DIR                     /dss/dsshome1/03/ga35kum3/SeisSol/submodules/yaml-cpp 
```

3. SuperMUC-NG, loaded modules:
```admin/1.0
tempdir/1.0
spack/release/19.1
lrz/default
scons/3.0.1
cmake/3.14.4
python/3.6_intel
slurm_setup/default
parmetis/4.0.3-intel-impi-i64-r64
metis/5.1.0-intel-i64-r64
netcdf/4.6.1-intel-impi-hdf5v1.8-parallel
libszip/2.1.1
numactl/2.0.11
hdf5/1.8.20-intel-impi-threadsafe
intel/19.0
mkl/2019
mpi.intel/2019
itac/2019
advisor_xe/2019
amplifier_xe/2019
inspector_xe/2019
devEnv/Intel/2019
```
4. Appended: scaling.sh, scaling.par, loh3.yaml

[scaling_5.sh.txt](https://github.com/SeisSol/SeisSol/files/3968219/scaling_5.sh.txt)
[scaling.par.txt](https://github.com/SeisSol/SeisSol/files/3968222/scaling.par.txt)
[loh3.yaml.txt](https://github.com/SeisSol/SeisSol/files/3968223/loh3.yaml.txt)

**Screenshots/Console output**
[SeisSol_LOH3_o6_n5.326157.out.txt](https://github.com/SeisSol/SeisSol/files/3968227/SeisSol_LOH3_o6_n5.326157.out.txt)

**Comment by uphoffc** [comment]
You could compile with make VERBOSE=1 and check whether the LOG_LEVEL, LOGLEVEL, and LOGLEVEL0 definitions are set.

**Comment by sebwolf-de** [comment]
These are the compiler options for the log levels, indeed strange:
-DLOGLEVEL0=2 -DLOGLEVEL0=3 -DLOGLEVEL=2 -DLOGLEVEL=3 -DLOG_LEVEL=2

**Comment by uphoffc** [comment]
Yes, should be -DLOGLEVEL=0 and -DLOG_LEVEL=0

**Comment by krenzland** [comment]
Fixed by #175
